File: program12/main.go
package main

import (
	"fmt"
)

func convertCurrency(amtInDls, conRate float64) float64 {
	conAmt := amtInDls * conRate
	fmt.Println(conAmt)
	return conAmt
}

func main() {
	var amt float64
	fmt.Print("Enter amount in $: ")
	fmt.Scan(&amt)

	var cr float64
	fmt.Print("Enter conversion rate: ")
	fmt.Scan(&cr)

	cnvrted := convertCurrency(amt, cr)
	fmt.Printf("₹ %.2f\n", cnvrted)
}

File: program12/main.py
def convert_currency(amt_in_dls,con_rate):
    con_amt=amt_in_dls*con_rate
    print(con_amt)
    return con_amt
amt=int(input("Enter amount in $: "))
cr=int(input("Enter conversion rate: "))
cnvrted=convert_currency(amt,cr)
print("₹",cnvrted)
